runner/src/server/plugins/engine/components/CheckboxesField.ts

```diff
@@ -31,6 +31,46 @@ export class CheckboxesField extends SelectionControlField {
       .join(", ");
   }
 
+  getAdditionalValidationFunctions(): Function[] {
+    return [
+      async (request, viewModel) => {
+        const checkboxesComponent = viewModel.components.find(
+          (c) => c.model.name === this.name
+        );
+        const payload = (request.payload || {}) as FormData;
+        let values = payload[this.name];
+
+        const componentKey = checkboxesComponent.model.id;
+
+        const error = {
+          path: componentKey,
+          name: componentKey,
+          href: `#${componentKey}`,
+        };
+
+        if (!this.options.behaviour) {
+          return [];
+        }
+
+        if (
+          values.some((value) => value === this.options.behaviourItemValue) &&
+          values.length > 1
+        ) {
+          return [
+            {
+              ...error,
+              ...{
+                text: `You cannot select ‘None of these’ and another option`,
```

Comment:
This text feels clunky, has this been provided? "You cannot select both 'None of these' and any other option". Not sure, it feels clunky to read

Reply:
Also agree, but assumed it was provided

Reply:
Yeah this was provided to us but them.